kitsune/users/views.py

```diff
@@ -152,7 +151,7 @@ def profile(request, username):
 @login_required
 @require_POST
 def close_account(request):
-    anonymize_user(request.user)
+    delete_user_pipeline(request.user)
 
     # Log the user out
     auth.logout(request)
```

Comment:
I believe you are automatically redirect and logged out on user deletion which means that the `auth.logout` is redundant but it's worth confirming it.

Reply:
As far as I can tell the logout and the redirect are not automatic, and have to be purposefully called.